ReleaseNotes.md

```diff
@@ -2,4 +2,6 @@
 * Expose the "next maintenance window" in the EMS system info.
 * Include support for the Admin User GetUsers route.
 # Release 2.4.1
-* Adds support for the new analytic set endpoints.
\ No newline at end of file
+* Adds support for the new analytic set endpoints.
+# Release 2.5.0
```

Comment:
I failed to mention this in a previous PR but this file should only contain the newest release (so delete the previous stuff). It gets attached as the text for the github release.

Reply:
ah I was wondering about that last time, I figured we just didn't have this before. Should be fixed now.